test/integration/targets/nxos_gir/tests/common/sanity.yaml

```diff
@@ -1,101 +1,110 @@
-#---
-#- debug: msg="START connection={{ ansible_connection }} nxos_gir sanity test"
+---
+- debug: msg="START connection={{ ansible_connection }} nxos_gir sanity test"
 - debug: msg="Using provider={{ connection.transport }}"
   when: ansible_connection == "local"
-#
+
+- set_fact: gir_run="true"
+- set_fact: gir_run="false"
+  when: platform is search("N35")
 #- name: "Setup"
 #  nxos_gir: &setup
 #    system_mode_maintenance: false
 #  ignore_errors: yes
-#
-#- block:
+
+- block:
+  - name: "Put system in maintenance mode with reload reset reason"
+    nxos_gir: &reset_reason
+      system_mode_maintenance_on_reload_reset_reason: manual_reload
```

Comment:
please add `provider: {{ connection }}` also in the testcases as we are restoring nxapi tests https://github.com/ansible/ansible/pull/41818

Reply:
Added provider